coursebuilder/views/base.html

```diff
@@ -235,226 +235,54 @@
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalgarage-lv">
-                Latvija
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalgarage-lt">
-                Lietuva
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digital-workshop-lu">
-                Luxembourg (EN)
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelierdigital-lu">
-                Luxembourg (FR)
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalworkshop-lu">
-                Luxemburg (DE)
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalworkshop-hu">
-                Magyarország
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalworkshop-mt">
-                Malta
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalewerkplaats">
-                Nederland
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalworkshop-at">
-                Österreich
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/internetowerewolucje">
-                Polska
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelierdigitalportugal">
-                Portugal
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelieruldigital">
-                România
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalakademin">
-                Sverige
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalnagaraza-si">
-                Slovenija
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalnagaraz">
-                Slovensko
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelierdigital-de">
-                Schweiz
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/digitalgarage-fi">
-                Suomi
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelierdigital-fr">
-                Suisse
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/atelierdigital-it">
-                Svizzera
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/dijitalatolye">
-                Türkçe
+                France
               </option>
 
               <option value="https://learndigital.withgoogle.com/digitalgarage" selected="">
                 United Kingdom
               </option>
```

Comment:
Probably just "Africa" and it points to `https://courses.academy.africa/` for now?

Reply:
@DavidLemayian Those are the languages (to be) supported by the site, see https://trello.com/c/VbYBu4ob/65-footer-on-main-page-home-page

Reply:
@andela-angene Actually you'll notice from the DigitalGarage site, the options are region not language. If you select a different region, it showcases different partners and yes, language does change as well.

Reply:
@DavidLemayian That's true, it's a nice effect. I added 'Africa' as the only option for now.